### DISCUSSION TITLE
[@types/formidable] - Fields and Files are now always arrays not strings.

### DISCUSSION THREAD

**Discussion by justin-harr** [DefinitelyTyped/DefinitelyTyped#66218] [Issues with a @types package] [answered]
As far as I can tell, after v3 release of [formidable](https://github.com/node-formidable/formidable) the Files and Fields returned from the parser will always be arrays.

So this:

```
    interface Fields {
        [field: string]: string | string[];
    }
    interface Files {
        [file: string]: File | File[];
    }
```

Should now be 
```
    interface Fields {
        [field: string]: string[];
    }
    interface Files {
        [file: string]: File[];
    }
```

Here is the changelog that announced the change - ["files and fields values are always arrays"](https://github.com/node-formidable/formidable/blob/master/CHANGELOG.md#300)

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment, marked as answer]
> 

PR is ready https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66332
